src/backend/Csrs.Api/Models/ModelExtensions.cs

```diff
@@ -221,6 +223,27 @@ public static Message ToMessage(MicrosoftDynamicsCRMssgCsrscommunicationmessage
             return value;
         }
 
+        private static async Task<int?> GetLookupValueAsync(IDynamicsClient dynamicsClient, string attributeName, string? label, CancellationToken cancellationToken)
+        {
+            if (label is null)
+            {
+                return null;
+            }
+
+            var metadata = await dynamicsClient.GetPicklistOptionSetMetadataAsync("ssg_csrsparty", attributeName,cancellationToken);
+            var values = metadata.ToViewModel();
+            var value = values.Where(_ => _.Value == label).SingleOrDefault();
+
+            if (value is null)
+            {
+                return null;
+            }
+            {
```

Comment:
What is this block for?  did you miss an `else`?

Reply:
Extra bracket was removed